src/notifications/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+cmake_minimum_required(VERSION 3.10)
+
+# set the project name
+project(Notification)
+
+set(NOTFYSRC
+  server.cpp
+  notify.cpp
+  client.cpp
+  )
```

Comment:
Didn't you say we didn't need this (and line 15) in this CMakeLists file?

Reply:
This is the CMake for making your own server and client, independent from the main make file. That's there for manual testing as the unit tests aren't working with sockets

Reply:
Yeah but yesterday didn't you say it worked without setting `NOTFYSRC` and you just needed the executables?

Reply:
Not in src/notifications/cmake list, perhaps I said it for a different one?

Reply:
The stuff we pair programmed didn't include this. This was something I made before that

Reply:
No, you added the `NOTFY` variable and the `add_library` section below when we did the pair programming. There's no point in doing this if you don't use it anywhere?

Reply:
Ah i think i understand the confusion now, you renamed from `LIBSRC` to `NOTFYSRC`. Even then, i don't think you need the `add_library` line

Reply:
I'll hope on discord